api/CookieStoreManager.json

```diff
@@ -0,0 +1,192 @@
+{
+  "api": {
+    "CookieStoreManager": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": "≤87"
```

Comment:
How do we end up with data like this? Was only Chrome for Android 87 tested, and it was supported there?

Reply:
That's correct, we only have results for Chrome Android 87.  I plan to obtain more results, however, as I mentioned during our meeting!